src/terminal/cli.rs

```diff
@@ -0,0 +1,40 @@
+use crate::types::nbodysystem::NBodySystem;
+use clap::Parser;
+
+/// N Body Problem
+#[derive(Parser, Debug)]
+#[command(version, about, long_about = None)]
+pub struct Args {
+    /// Print the velocities and coordinates [none, plain_text, csv, json]
+    #[arg(short, long, default_value = "none")]
+    output: String,
+
+    /// Limit the output of information about particles to once every N steps
+    #[arg(short, long, default_value_t = 1000)]
+    limit: u16,
+
+    /// Print text to a file instead of the console
+    #[arg(short, long)]
+    file: bool,
```

Comment:
use_file_output instead of file for this variable name seems to be better, since it's a boolean.